docs/learning-powershell/README.md

```diff
@@ -119,9 +118,8 @@ Note that all bash commands should continue working on PowerShell session.
 [getstarted-with-powershell]: https://channel9.msdn.com/Series/GetStartedPowerShell3
 [why-learn-powershell]: https://blogs.technet.microsoft.com/heyscriptingguy/2014/10/18/weekend-scripter-why-learn-powershell/
 [Using Windows PowerShell for Administration]: https://docs.microsoft.com/powershell/scripting/getting-started/fundamental/using-windows-powershell-for-administration?view=powershell-6
-[ebook-from-powershell.com]: http://powershell.com/cs/blogs/ebookv2/default.aspx
```

Comment:
Looks like the powershell.com website is broken. These types of issues should not result in a static analysis failure. They should generate a warning to draw attention to the problem, but they shouldn't block all CI builds from happening like they are doing right now. How about coming up with a way to make certain static analysis failures generate warnings so that the issues are discovered without blocking builds?

That aside, while powershell.com is down right now, I believe you can just link to idera directly for those links.

The eBook is here: https://www.idera.com/resourcecentral/whitepapers/powershell-ebook
The Video Library is here: https://community.idera.com/database-tools/powershell/video_library/

Reply:
Thanks.  I'll update to those links.